app/models/user.rb

```diff
@@ -2,6 +2,8 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: :author_id
   has_many :comments, foreign_key: :author_id
   has_many :likes, foreign_key: :author_id
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
Please note posts_counter must be an integer greater than zero only.

- [ ] Kindly update this validation

Reply:
Done✔